README.md

```diff
@@ -1,6 +1,8 @@
 Post JIRA comment with build details
 ============================================
 
+We have moved this step to https://github.com/bitrise-community/post-jira-comment-with-build-details-step repository. It has been rewritten in Golang.
```

Comment:
Changelog on https://discuss.bitrise.io/t/post-jira-comment-v1-0-0/4962/1 points to https://github.com/bitrise-steplib/steps-post-jira-comment-with-build-details/ as a new location. `bitrise-steplib` org not `bitrise-community`. community is the current location.